### PR TITLE
3.1

### PR DESCRIPTION
**Pull request by azazellochg**
micassess updated to 1.0.0, pep8 fixes

### CODE REVIEW

File: cryoassess/protocols/protocol_micassess.py [hunk 1]
```diff
@@ -59,8 +59,13 @@ def _defineParams(self, form):
                       pointerClass='SetOfMicrographs',
                       label="Input micrographs", important=True)
         form.addParam('threshold', params.FloatParam, default=0.1,
-                      label='Threshold',
-                      help='Threshold for classification. Default is 0.1. '
+                      label='Threshold 1',
+                      help='Threshold for good/bad classification. Default is 0.1. '
+                           'Higher number will cause more good micrographs '
+                           'being classified as bad.')
+        form.addParam('threshold2', params.FloatParam, default=0.1,
```
Comment (DaniDelHoyo) [inline cryoassess/protocols/protocol_micassess.py:66]: Is threshold2 based on the same score as threshold1? In that case, I'm guessing it should always be higher than threshold1, should we add a warning if this is not true?. And in any case, could you elaborate a bit this on the help? Now it is the same as in threshold1.

**Comment by DaniDelHoyo** [comment]
Apart from the bug in https://github.com/scipion-em/scipion-em-cryoassess/pull/15 (which was probably found in previous versions too) I run some tests and everything seems to be working properly

File: cryoassess/protocols/protocol_micassess.py (hunk 1 above)
Comment (azazellochg) [reply cryoassess/protocols/protocol_micassess.py:66]: Hi @danihr , 
"The prediction uses a hierarchical classification model. For a new micrograph, the model will first decide whether this is a "good" or "bad" micrograph (validation accuracy ~93%). The tolerance threshold in this step can be tuned by the user with --t1. If the micrograph is classified as "good", the model will further classify whether it belong to the "great" class or "decent" class (validation accuracy ~75%). The tolerance threshold in this step can be tuned by the user with --t2. If the micrograph is "bad", the model will further classify which "bad" class it belongs to (validation accuracy ~80%)."

**Comment by azazellochg** [comment]
@delarosatrevin was mentioning that it does not work in streaming. I havent checked this.

**Comment by DaniDelHoyo** [comment]
> @delarosatrevin was mentioning that it does not work in streaming. I havent checked this.

I did check it and it was working for me.

**Comment by azazellochg** [comment]
Great, thanks for revieweing this!

**Comment by DaniDelHoyo** [comment]
> > @delarosatrevin was mentioning that it does not work in streaming. I havent checked this.
> 
> I did check it and it was working for me.

just checked again and with several threads and it still works, so unless @delarosatrevin comment any other bug, I would approve this

**Comment by DaniDelHoyo** [comment]
> Great, thanks for reviewing this!

you are welcome, any time

**Review by DaniDelHoyo — APPROVED** [review]

**Pull request merged**